core/src/main/java/com/netflix/msl/io/DefaultMslEncoderFactory.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) 2017 Netflix, Inc.  All rights reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.msl.io;
+
+import java.io.InputStream;
+import java.util.Set;
+
+/**
+ * <p>Default {@link MslEncoderFactory} implementation that supports the
+ * following encoder formats:
+ * <ul>
+ * <li>JSON: backed by {@code org.json}.</li>
```

Comment:
This specific default implementation is backed by org.json because it explicitly uses classes that use org.json. A different MslEncoderFactory concrete subclass, say NetflixMslEncoderFactory, might support JSON backed by a different library like Jackson or GSON.